01_02_Combat_Rules.txt

```diff
@@ -186,6 +186,26 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
 	A target reduced to exactly 1 health by a nonlethal attack does not 
 fall Unconscious. 1 is not 0.
 
+Melee Accuracy and Guard Critical Hits:
+	See "Critical Hits" below for general Critical information. Melee Critical 
+hits deal extra damage as normal. Critical Guard rolls block all damage from any 
+non-Extra-critical melee attack. Extra-critical Guard rolls block all damage from 
+the incoming attack. 
+
+	Melee Accuracy rolls may Critically fail in the same way as firearms. 
+Unless otherwise specified, melee weapons have a Jam range of 2. Should your 
+melee weapon "jam", it catches on something in the environment at the DM's 
+discretion, if applicable, and is stuck. Unsticking the weapon requires a 
+Weapons - Melee check as a firearm would, except that if you pass the DC by 
+at least 30 you do not provoke an Opportunity Attack. Should nothing be 
+available for the weapon to get stuck in, it is instead dropped, but the 
+skill check may still be used to attempt to avoid an Opportunity Attack for 
+picking it up (you will always succeed in picking it up if it's still within 
+reach).
```

Comment:
For a cleaner sentence with a less informal tone: "...dropped. You will always succeed in picking up a dropped weapon if it is within reach, but the skill check may still be used to attempt to avoid an Opportunity Attack while picking it up.